front/app/components/ProjectDescriptionBuilder/Settings/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import { useEditor, ROOT_NODE } from '@craftjs/core';
+
+import { CONTENT_BUILDER_DELETE_ELEMENT_EVENT } from 'components/admin/ContentBuilder/constants';
+
+import { MessageDescriptor } from 'utils/cl-intl';
+import eventEmitter from 'utils/eventEmitter';
+
+import Settings from '../../admin/ContentBuilder/Settings/Settings';
+import { SelectedNode } from '../../admin/ContentBuilder/Settings/typings';
+
+const ContentBuilderSettings = () => {
```

Comment:
This component was previously shared between the project description builder and the homepage builder. The homepage builder now uses a different version that is shared with the report builder, so now the project description builder is the only one using this legacy settings object.

The reason why we changed it for the homepage builder (and the report builder before) is because we don't store this data inside of the craftjs custom attribute anymore.